tests/tailer/savedown_withcount/tests.csv

```diff
@@ -0,0 +1,16 @@
+action,ms,bytes,lang,code,repeate,minver,comment
+before,0,0,q,tr:.servers.gethandlebytype[`tr_seg1;`any],1,,"Connect to tailreader handle to give access the tailreader1 for testing tailer1"
+before,0,0,q,tl:.servers.gethandlebytype[`tailer_seg1;`any],1,,"Connect to tailer1 handle,so tailer can be tested through tl handle"
+before,0,0,q,tl".wdb.numrows:100",1,,"Set row count as the standard row count is 10000 would be to large for a test"
+run,0,0,q,(tr"count tables[]")=2,1,,"Check the number of tables in the tailreader is currently 2"
+run,0,0,q,tl".ds.savealltablesoverperiod[]",1,,"Running savedown function sould save tables but, there should be no data to savedown so the table count should stay the same"
```

Comment:
if I'm not mistaken, this function takes two parameters? how does this test work if that's the case?